integration-tests/src/lib.rs

```diff
@@ -112,11 +116,9 @@ fn para_to_system_para_sender_assertions(t: ParaToSystemParaTest) {
 	assert_expected_events!(
 		PopNetworkPara,
 		vec![
-			// Amount to reserve transfer is transferred to Parachain's Sovereign account
-			RuntimeEvent::Balances(
-				pallet_balances::Event::Withdraw { who, amount }
-			) => {
-				who: *who == t.sender.account_id,
+			// Amount to teleport is withdrawn from Sender
+			RuntimeEvent::Balances(pallet_balances::Event::Burned { who, amount }) => {
+				who: *who == t.receiver.account_id,
```

Comment:
```suggestion
				who: *who == t.sender.account_id,
```

This is wrong and needs to be reverted.